packages/fileeditor-extension/schema/plugin.json

```diff
@@ -9,6 +9,12 @@
         "autoClosingBrackets": {
           "type": "boolean"
         },
+        "fontFamily": {
```

Comment:
permitting this to be `null` and handling that explicitly, with some doc suggesting that it will delegate to the CSS theme if left blank might be more robust... just deleting the key might not be sufficient if the style is already set, for example.